testing/camunda-process-test-java/src/main/java/io/camunda/process/test/api/BpmnAssert.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.process.test.api;
+
+import io.camunda.process.test.api.assertions.ProcessInstanceAssert;
+import io.camunda.process.test.impl.assertions.CamundaDataSource;
+import io.camunda.process.test.impl.assertions.ProcessInstanceAssertj;
+import io.camunda.zeebe.client.api.response.ProcessInstanceEvent;
+import io.camunda.zeebe.client.api.response.ProcessInstanceResult;
+import java.time.Duration;
+import org.awaitility.Awaitility;
+
+public class BpmnAssert {
```

Comment:
❓ I know you aim to keep the API aligned with ZPT as much as possible. However, I wonder if it makes sense to rename this class to `CamundaAssert`. It aligns more with the naming of the rest of the project. I'm also thinking about what if we add DMN support in the future, then the name doesn't make sense. We'd have to add a `DmnAssert`.
On the flip-side, maybe adding DMN doesn't make sense anyways considering the project is called Camunda **process** test.

Reply:
Interesting idea. I'll rename it. :+1: